engine/frontend/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
+    "@testing-library/jest-dom": "^5.16.5",
```

Comment:
are all these dependencies necessary for prod? just checking

Reply:
probably not; but can remove the un-needed ones in upcoming PR(s). I just kept testing ones as they may get handy for me to add some tests later on